src/AddAttributesTwigExtension.php

```diff
@@ -50,11 +50,8 @@ public function add_attributes($context, $additional_attributes = []) {
           if ($value instanceof Attribute) {
             $value = $value->toArray()[$key];
           }
-          elseif (is_string($value)) {
-            $value = [$value];
-          }
           else {
-            continue;
+            $value = [$value];
```

Comment:
@callinmullaney I am thinking this is a little too open, it seems like your use case (original question) is to support integers.  Should we not have another case to handle integers (convert them to a string)?

Currently this is what I am seeing we are processing:
1. Arrays (flat) - technically nested arrays would just not work
2. (object) Attribute
3. Strings
4. Default - Do nothing

I think this would read easier if we were using a switch statement here.

Reply:
@ccjjmartin I updated this PR to use a switch statement like you suggested. The original scope of this issue was to handle passing integers type values but with the changes I made we can also handle boolean. 

That being said, while nested arrays and objects would also be a nice thing to process if passed to `add_atributes()` I think it's a bit out of scope for the original issue. The switch statement framework established here would be a good jumping off point for another branch to add support.

Can you take a quick look at this and confirm the testing criteria above?